bundle/manifests/opentelemetry.io_instrumentations.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.6.0-beta.0
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.17.0/#gov3-upgrade-go-and-dependencies